src/scripts/choices.js

```diff
@@ -177,6 +177,14 @@ class Choices {
     this.init();
   }
 
+  /* ========================================
+  =            Static properties            =
+  ======================================== */
+
+  static get Templates() {
+    return TEMPLATES;
+  }
```

Comment:
👍 Can we define `userDefaults` here too please? See https://github.com/jshjohnson/Choices/blob/master/src/scripts/choices.js#L2095 

Is there a reason for this property not being upper cased?

Reply:
`userDefaults` are mutable, so, we will need to introduce extra variable to keep it value. Are you ok with that? It seems more logical to have it as static property, but.

I don't know your conventions for this lib regarding static properties, what would you prefer - Choices.TEMPLATES or Choices.TEMPLATES? I think it should be more or less in line with userDefaults?

Reply:
I definitely like the form of `Choices.defaults.xyz` 👍

Reply:
This will be going into the next major release anyway so we can have breaking changes here